dxa-framework/dxa-common-impl/src/main/java/com/sdl/webapp/common/impl/localization/LocalizationFactoryImpl.java

```diff
@@ -120,21 +119,34 @@ private void loadMainConfiguration(String id, String path, LocalizationImpl.Buil
                 });
     }
 
-    private void loadVersion(String id, String path, LocalizationImpl.Builder builder)
-            throws LocalizationFactoryException {
+    private Boolean loadVersionFromProperties(String id, String path, LocalizationImpl.Builder builder)
```

Comment:
LocalizationFactoryImpl is a Spring bean, use @Value annotation instead of manual loading of properties.